vllm/platforms/openvino.py

```diff
@@ -1,9 +1,12 @@
 from typing import TYPE_CHECKING
 
+import openvino as ov
+import openvino.properties.hint as hints
 import torch
 
 import vllm.envs as envs
 from vllm.logger import init_logger
+from vllm.utils import GiB_bytes
```

Comment:
I think you need to import it lazily inside the function

Reply:
fixed in [4e5c594](https://github.com/vllm-project/vllm/pull/10573/commits/4e5c594644bdd8584fd8ebf83a83d55908ada5ae)

Reply:
PTAL@youkaichao